PowerFGT/Public/Connection.ps1

```diff
@@ -168,6 +168,17 @@ function Connect-FGT {
         #Add csrf cookie to header (X-CSRFTOKEN)
         $headers = @{"X-CSRFTOKEN" = $cookie_csrf }
 
+        $uri = $url + "logindisclaimer"
+        if($iwrResponse.Content -match '/logindisclaimer')
+        {
+            try {
+                Invoke-WebRequest $uri -Method "POST" -WebSession $FGT @invokeParams -Body @{confirm = 1;ajax=1} | Out-Null
```

Comment:
missing @invokeParams ?
add space after ; (on body) =>  @{confirm = 1; ajax = 1 }

Reply:
I'm not sure I understand the change you're requesting.

invokeParams is between the websession and body parameters.

Reply:
Space added in Body parameter before "ajax".